test/evaluate.js

```diff
@@ -0,0 +1,100 @@
+import { evaluate } from '../lib/evaluate.js'
+import assert from 'node:assert'
+import { ethers } from 'ethers'
+
+const { BigNumber } = ethers
+
+describe('evaluate', () => {
+  it('evaluates measurements', async () => {
+    const rounds = { 0: [] }
+    for (let i = 0; i < 10; i++) {
+      rounds[0].push({ peerId: '0x123' })
+    }
+    const ieContractWithSigner = {
+      async setScores (roundIndex, peerIds, scores, summary) {
+        assert.strictEqual(roundIndex, 0)
+        assert.deepStrictEqual(peerIds, ['0x123'])
+        assert.strictEqual(scores.length, 1)
+        assert.strictEqual(
+          scores[0].toString(),
+          BigNumber.from(1_000_000_000_000_000).toString()
+        )
+        assert.match(summary, /^\d+ retrievals$/)
```

Comment:
Would you mind applying the same approach as I described in https://github.com/filecoin-station/spark-api/pull/63#discussion_r1314694193 and move the assertions to the end of the test after we call `evaluate()`?

The same comment applies to other similar tests in this PR too.